pyproject.toml

```diff
@@ -31,101 +31,102 @@ include = [
 # Background task processing
-celery = [
-    {version = "~5.2.2", python = "^3.7"},
-    {version = "~5.1.2", python = "<3.7"},
-]
+celery = "~5.2.3"
 # Fundamental web framework for Nautobot
-Django = "~3.1.12"
+Django = "~3.2"
 # Adds AJAX capabilities to django-tables2
 django-ajax-tables = "~1.1.1"
 # LDAP Support
-django-auth-ldap = {version = "~3.0.0", optional = true}
+django-auth-ldap = {version = "~4.0.0", optional = true}
 # Caching with Redis
-django-cacheops = "~5.1"
+django-cacheops = "~6.0"
 # Base functionality for task scheduling with Celery
-django-celery-beat = "^2.2.0"
+django-celery-beat = "~2.2.1"
 # Management of app configuration via the Django admin UI
 django-constance = {version = "~2.8.0", extras = ["database"]}
 # Permit cross-domain API requests
-django-cors-headers = "~3.7.0"
+django-cors-headers = "~3.10.1"
 # Support for encrypted database fields
 django-cryptography = "~1.0"
 # Store files in the database for background tasks
 django-db-file-storage = "~0.5.5"
 # AutoSlugField and shell_plus
 django-extensions = "~3.1.5"
 # Advanced query filters
-django-filter = "~2.4.0"
+django-filter = "~21.1"
 # Health check endpoint
-django-health-check = "^3.16.4"
+django-health-check = "~3.16.5"
 # Django Jinja - jinja2 backend for django, used to easily register custom jinja filters and render jinja.
-django-jinja = "<2.8.0"
-# Modified Preorder Tree Traversal - tree structure for Region, RackGroup, etc.
-django-mptt = "~0.11.0"
+django-jinja = "~2.10.0"
+# Modified Pre-order Tree Traversal - tree structure for Region, RackGroup, etc.
+django-mptt = "~0.13.4"
 # Prometheus metrics for Django
-django-prometheus = "~2.1.0"
+django-prometheus = "~2.2.0"
 # Redis cache for Django used for distributed locking
 django-redis = "~5.2.0"
-# RQ (Redis Queueing) for background handlin of webhooks, jobs, etc.
-django-rq = "~2.4.1"
+# RQ (Redis Queueing) for background handling of webhooks, jobs, etc.
+django-rq = "~2.5.1"
 # External Storage support (i.e. S3)
 django-storages = {version = "~1.12.3", optional = true}
 # Advanced HTML tables
-django-tables2 = "~2.3.4"
+django-tables2 = "~2.4.1"
 # Tags
-django-taggit = "~1.3.0"
+# NOTE: django-taggit 2.x is available but has breaking API changes
+django-taggit = "~1.5.1"
 # Represent time zones in Django
+# NOTE: django-timezone-field 4.2.x is available but appears to break our initial migrations?
 django-timezone-field = "~4.1.2"
 # Run production webservers such as uWSGI/gunicorn as a Django management command.
 django-webserver = "~1.2.0"
 # REST API framework
-djangorestframework = "~3.12.4"
-# Swagger schema generation for the REST API
-drf-yasg = {extras = ["validation"], version = "~1.20.0"}
+djangorestframework = "~3.13.1"
+# OpenAPI 3.0 schema generation for the REST API
+# TODO: fix for #688 is in master, not yet released
+drf-spectacular = {git = "https://github.com/tfranzel/drf-spectacular.git", rev = "6f5bd1688182829204bb61db6a9d7903fdaf0bbe", extras = ["sidecar"]}
```

Comment:
Still waiting for a new release of drf-spectacular so that we don't need to pin to a commit hash - should be any day now.